src/page/index.html

```diff
@@ -1,32 +1,340 @@
+            <p>そうだ。<br />カフェに行こう。</p>
+        </div>
+        <div class="message_outer">
+            <div class="message_title">
+                カフェで読書＆コワーキング。快適のフロアスペース空間。
+            </div>
+            <div class="message_content">
+                カフェテリアでコーヒーを注文し、ブックエリアでお好みの本を選び、コワーキングスペースでコーヒーとコーヒーと共に読書を楽しむ。<br />学校、仕事帰りに、平日の日中に、休日にゆっくり...さまざまなライフスタイルに合わせてご利用いただける多彩なプランをご用意しております。
+            </div>
+        </div>
+    </div>
+    <div class="container-fluid about">
+        <div class="img-grid">
+            <div class="img-item1"></div>
+            <div class="img-item2"></div>
+            <div class="img-item3"></div>
+            <div class="img-item4"></div>
+        </div>
+
+        <div class="message_outer">
+            <div class="message_title">
+                人と人、人と本が繋がるまったく新しい快適時間。
+            </div>
+            <div class="message_content">
+                月額プラン以外にビジター料金でのご利用も可能です。通いやすい時間や曜日に合わせてプランをお選びください。 ご登録プランは後の変更も可能ですので、資格試験等の目標がある場合など環境の変化に合わせて柔軟に対応しております。WiFiも電源も標準設 備しており、 時間を気にせず仕事や勉強に取り組めます。カフェでの長居は気が引けるという方にも心置きなくご利用いただけます。
+            </div>
+        </div>
+    </div>
+    <div class="cards container">
+        <div class="card">
+            <div class="card-img">
+                <div class="img-flg"></div>
+                <p class="card-title">CONCEPT</p>
+                <img src="./img/card-1.jpg" class="card-img-top" alt="" />
+            </div>
+            <div class="card-body">
+                <p class="card-text">
+                    「本屋」でも「自習室」でもない、新しい空間。それがこのカフェ＆コワーキングです。お好きなドリンクを飲みながらゆったりと気兼ねなく勉強や仕事に取り組んでいただけます。
+                </p>
+                <div class="card-button">
+                    <a href="#" class="button">コンセプト</a>
+                </div>
+            </div>
+        </div>
+        <div class="card">
+            <div class="card-img">
+                <div class="img-flg"></div>
+                <p class="card-title">SERVICE</p>
+                <img src="./img/card-2.jpg" class="card-img-top" alt="" />
+            </div>
+            <div class="card-body">
+                <p class="card-text">
+                    喫茶・軽食を楽しめるカフェスペース。ビジネス書から最新の雑誌まで取りそろえるブックスペース。ブース型のパーソナルワークスペースに、開放感があるコワーキングスペース。
+                </p>
+                <div class="card-button"><a href="#" class="button">サービス</a></div>
+            </div>
+        </div>
+        <div class="card">
+            <div class="card-img">
+                <div class="img-flg"></div>
+                <p class="card-title">FLOOR</p>
+                <img src="./img/card-3.jpg" class="card-img-top" alt="" />
+            </div>
+            <div class="card-body">
+                <p class="card-text">
+                    カフェ・ブック・コワーキング。各エリアで異なるコンセプトを持つ当店では、目的や気分に合わせてフロア内を移動することで充実した時間をお過ごしいただけます。
+                </p>
+                <div class="card-button"><a href="#" class="button">フロアー</a></div>
+            </div>
+        </div>
+    </div>
+    <div class="container-xxl news-and-events">
```

Comment:
```suggestion
    <div class="news-and-events">
      <div class="container">
```
こんな感じで2つの要素に分割し、 `news-and-events` の方に背景色を設定してあげると、左右の余白はcontainer準拠で背景色は画面幅いっぱいに広がるという挙動が実現できるはずです。